files/fr/web/performance/lazy_loading/index.html

```diff
@@ -1,40 +1,40 @@
 
 <h2 id="general_overview">Vue d'ensemble</h2>
 
-<p>Au fur et à mesure de l'évolution du web, nous avons vu une grande augmentation du nombre et de la taille des ressources servie aux internautes. Entre 2011 et 2019, le poids médian des ressources est passé de <strong>~100&nbsp;Ko</strong> à <strong>~400&nbsp;Ko</strong> sur ordinateur et de <strong>~50&nbsp;Ko</strong> à <strong>~350&nbsp;Ko</strong> sur mobile. Tandis que la taille médiane des images est passée de <strong>~250&nbsp;Ko</strong> à <strong>~900&nbsp;Ko</strong> sur ordinateur et de <strong>~100&nbsp;Ko</strong> à <strong>~850&nbsp;Ko</strong> sur mobile.</p>
+<p>Au fur et à mesure de l'évolution du web, nous avons vu une grande augmentation du nombre et de la taille des ressources servie aux internautes. Entre 2011 et 2019, le poids médian des ressources est passé de <strong>~10 Ko</strong> à <strong>~40 Ko</strong> sur ordinateur et de <strong>~5 Ko</strong> à <strong>~35 Ko</strong> sur mobile. Tandis que la taille médiane des images est passée de <strong>~25 Ko</strong> à <strong>~90 Ko</strong> sur ordinateur et de <strong>~10 Ko</strong> à <strong>~85 Ko</strong> sur mobile.</p>
 
-<p>L'une des méthodes utilisables pour s'occuper de ce problème consiste à réduire la longueur du <a href="/fr/docs/Web/Performance/Critical_rendering_path">chemin critique de rendu</a> en chargeant les ressources de façon différée lors du premier rendu de la page. Exemple concret : lorsque vous arrivez sur la page d'accueil d'une boutique en ligne disposant d'un lien vers la section « panier », il est plus optimal de ne charger ses ressources que lorsque vous naviguerez vers cette section.</p>
+<p>L'une des méthodes utilisables pour s'occuper de ce problème consiste à réduire la longueur du <a href="/fr/docs/Web/Performance/Critical_rendering_path">chemin critique de rendu</a> en chargeant les ressources de façon différée lors du premier rendu de la page. Exemple concret : lorsque vous arrivez sur la page d'accueil d'une boutique en ligne disposant d'un lien vers la section « panier », il est plus optimal de ne charger ses ressources que lorsque vous naviguerez vers cette section.</p>
 
 <h2 id="strategies">Stratégies</h2>
 
 <p>Le chargement différé peut être appliqué sur de multiples ressources et avec de multiples stratégies.</p>
 
 <h3 id="general">En général</h3>
 
-<p id="code_splitting"><strong>Division du code&nbsp;:</strong><br>
-Le code JavaScript, CSS et HTML peuvent être divisés en petit morceaux. Cela permet de n'envoyer que la portion de code nécessaire à l'affichage sur l'écran de l'internaute, et donc d'améliorer les temps de chargement. Le reste sera chargé sur demande. Deux systèmes sont possibles&nbsp;:</p>
+<p id="code_splitting"><strong>Division du code :</strong><br>
```

Comment:
@tristantheb non-breaking spaces are needed here to avoid orphaned ponctuation on small screens.